css/components/updates.css

```diff
@@ -0,0 +1,96 @@
+section.updates > .row > .col-xs-12 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  background-color: #fff;
+  border-radius: 10px;
+  box-shadow: 0 0 20px 0 #0000002d;
+  padding: 50px 0;
+  margin-top: 100px;
+}
+
+section.updates h2 {
+  text-transform: capitalize;
+}
+
+.flaticon-mail-1::before {
+  font-size: 230px;
+  color: #01a0f9;
+  background: -webkit-linear-gradient(#01a0f9, #09c778);
+  -webkit-background-clip: text;
+  background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
+
+.updates-left-side {
+  display: flex;
+  justify-content: center;
+  padding: 0 40px;
+}
+
+.updates-right-side {
+  padding: 0;
+}
+
+.updates-form {
+  margin-top: 30px;
+  position: relative;
+  width: 70%;
+}
+
+.updates-form input {
+  border: 0;
+  width: 100%;
+  border-bottom: 1px solid #01a0f9;
+  font-size: 14px;
+  line-height: 35px;
+  outline: none;
+}
+
+.updates-form input::placeholder {
+  text-transform: capitalize;
+}
+
+.updates-form input:focus {
```

Comment:
There's no box-shadow in original Neuron page input part.

Reply:
in the original there is nothing on `:focus` state, which is not good, so I put box-shadow on it to be noticable if you navigate the page only with keyboard.

Reply:
Okey :)